src/google-analytics.js

```diff
@@ -92,15 +92,17 @@ export default class GoogleAnalytics extends BasePlugin {
    */
   _init(): void {
     if (!window.google_tag_manager) {
-      Utils.Dom.loadScriptAsync(`${GoogleAnalytics.GTAG_LIB_URL}?id=${this.config.trackingId}`).then(() => {
+      Utils.Dom.loadScriptAsync(`${GoogleAnalytics.GTAG_LIB_URL}?id=${this.config.trackingId || this.config.trackingGA4Id}`).then(() => {
```

Comment:
If both are configured is it ok to send only the old tracking id and not the GA4?

Reply:
yes. see https://developers.google.com/analytics/devguides/collection/ga4/basic-tag?technology=gtagjs